packages/components/carousel/src/carousel-control.tsx

```diff
@@ -78,6 +77,7 @@ const CarouselControl = forwardRef<
       className={cx("ui-carousel__control", className)}
       colorScheme={colorScheme}
       isRounded
+      aria-label={`Go to ${operation} slide`}
```

Comment:
This would cause "Go to previous slide" to become "Go to prev slide," so it needs to be redesigned.